frontend/public/components/notification-drawer.tsx

```diff
@@ -70,6 +70,14 @@ const AlertEmptyState: React.FC<AlertEmptyProps> = ({ drawerToggle }) => (
   </EmptyState>
 );
 
+const actionAlerts: ActionAlertType[] = [
+  {
+    id: '3929940202',
```

Comment:
Is this `id` going to be stable? I'd suggest looking for `AlertmanagerReceiversNotConfigured` instead